lib/ChangeTypes.js

```diff
@@ -67,6 +67,8 @@ module.exports = {
   MULTIPLY_COEFFICIENTS: 'MULTIPLY_COEFFICIENTS',
   // e.g. 2x * x -> 2x ^ 2
   MULTIPLY_POLYNOMIAL_TERMS: 'MULTIPLY_POLYNOMIAL_TERMS',
+  // e.g. nthRoot(x, 2) * nthRoot(x, 2) -> nthRoot(x^2, 2)
+  MULTIPLY_NTH_ROOTS: 'MULTIPLY_NTH_ROOTS',
```

Comment:
I think this should go in the Roots section, rather than the Multiplying Polynomials section.